com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Textures/leCat_tex.png.meta

```diff
@@ -0,0 +1,98 @@
+fileFormatVersion: 2
```

Comment:
Do we need this texture to be 2048x2048? I wonder if we could get away with 1024x1024 to reduce package size? The mip map debug mode hints we might be able to reduce: 

![image](https://user-images.githubusercontent.com/13305729/175578435-1718bf74-e4d0-4c33-9f58-eccf059dd68a.png)